ape-config.yaml

```diff
@@ -35,90 +33,6 @@ solidity:
     - "@threshold/contracts=threshold/v1.2.1"
 
-deployments:
-  polygon:
-    mainnet:
-      - contract_type: DAI
-        address: "0x8f3Cf7ad23Cd3CaDbD9735AFf958023239c6A063"
-      - contract_type: fx_child
-        address: "0x8397259c983751DAf40400790063935a11afa28a"
-    mumbai:
-      - contract_type: DAI
-        address: "0x001B3B4d0F3714Ca98ba10F6042DaEbF0B1B7b6F"
-      - contract_type: StakeInfo
-        address: "0xC1379866Fb0c100DCBFAb7b470009C4827D47DD8"
-      - contract_type: fx_child
-        address: "0xCf73231F28B7331BBe3124B907840A94851f9f11"
-      - contract_type: TACoChildApplication
-        address: "0x68E95C2548363Bf5856667065Bc1B89CC498969F"
-  ethereum:
-    local:
-      - nu_token_supply: 1_000_000_000
-        pre_min_authorization: 40000000000000000000000
-        pre_min_operator_seconds: 86400 # one day in seconds
-        pre_hash_algorithm: 1
-        pre_base_penalty: 2
-        pre_penalty_history_coefficient: 0
-        pre_percentage_penalty_coefficient: 100000
-        reward_duration: 604800 # one week in seconds
-        deauthorization_duration: 5184000 # 60 days in seconds
-        commitment_duration_1: 15552000 # 180 days in seconds
-        commitment_duration_2: 31104000 # 360 days in seconds
-        verify: False
-    rinkeby:
-      - nu_token: "0x78D591D90a4a768B9D2790deA465D472b6Fe0f18"
-        nu_token_supply: 1_000_000_000
-        t_staking: "0x18eFb520dA5D387982C860a64855C14C0AcADF3F"
-        work_lock: "0x0000000000000000000000000000000000000000"
-        staking_escrow: "0x6A6F917a3FF3d33d26BB4743140F205486cD6B4B"
-        pre_min_authorization: 40000000000000000000000
-        pre_min_operator_seconds: 86400 # one day in seconds
-        verify: True
-    # TODO: is there a batter way to manage multiple deployments on the same network?
-    # goerli-tapir:
-    #   - t_staking: '0x09B5065d2924De33C85F76474A89A27189402064'
-    #     pre_min_authorization: 40000000000000000000000
-    #     pre_min_operator_seconds: 86400  # one day in seconds
-    #     verify: True
-    # goerli-oryx:
-    #   - t_staking: '0xf91afFE7cf1d9c367Cb56eDd70C0941a4E8570d9'
-    #     verify: True
-    #     pre_min_authorization: 40000000000000000000000
-    #     pre_min_operator_seconds: 86400  # one day in seconds
-    goerli: # -lynx
-      - t_staking: "0x81eEefb7B1b3313C89976910096F8f1487301Bb1"
-        pre_min_authorization: 40000000000000000000000
-        pre_min_operator_seconds: 86400 # one day in seconds
-        ritual_timeout: 3600
-        max_dkg_size: 8
-        pre_application: "0x685b8Fd02aB87d8FfFff7346cB101A5cE4185bf3"
-        verify: True
-        reward_duration: 604800 # one week in seconds
-        deauthorization_duration: 5184000 # 60 days in seconds
-      - contract_type: fx_root
-        address: "0x3d1d3E34f7fB6D26245E6640E1c50710eFFf15bA"
-      - contract_type: checkpoint_manager
-        address: "0x2890bA17EfE978480615e330ecB65333b880928e"
-    mumbai:
```

Comment:
Add to PR description that this PR fixes #88

Reply:
✔️